RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py

```diff
@@ -62,3 +62,14 @@
     OrderedHitsFactoryPSet = dict(maxElement = 100000),
     RegionFactoryPSet = dict(RegionPSet = dict(ptMin = 0.3)),
 )
+eras.trackingPhase2PU140.toModify(photonConvTrajSeedFromSingleLeg,
+    ClusterCheckPSet = dict(
+        MaxNumberOfCosmicClusters = 1000000,
+        MaxNumberOfPixelClusters = 100000,
+        cut = None
+    ),
+    OrderedHitsFactoryPSet = dict(maxElement = 100000),
+    RegionFactoryPSet = dict(RegionPSet = dict(ptMin = 0.3)),
+    primaryVerticesTag   = cms.InputTag("offlinePrimaryVertices"),
```

Comment:
I meant that `primaryVerticesTag` is already set to `offlinePrimaryVertices` at line 6, and hence this line doesn't do anything and could be removed.